### DISCUSSION TITLE
Package submissions

### DISCUSSION THREAD

**Discussion by pmig** [glasskube/glasskube#90] [Announcements]
# Add your package to the Glasskube package manager

🧑‍🎨 You built a cloud native application that can be deployed on Kubernetes?

🙏 Please add it to Glasskube—it only takes a few seconds! ⏱

### 1. Just post a comment below in **this** GitHub discussion with some basic information:

- **Package name**:
- **Short description**:
- **Website URL**: 
- **Source code URL**: 
- **Logo URL**: 
- **Preferred installation method**:  manifest / helm 
- **Tags**: see below the list of available tags

### 2. If you want to list your package on our [website](https://glasskube.dev/packages/), you can already create a pull request: 

https://github.com/glasskube/glasskube/blob/main/website/src/data/packages.tsx

#### Tag list

- ai
- backup
- configuration
- database
- delivery
- logging
- monitoring
- networking
- security
- visualization

**Comment by asaiacai** [comment]
- Package name: gpu-operator
- Short description: deploys NVIDIA gpu drivers, container runtime and exporters
- Website URL: https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/index.html
- Source code URL: https://github.com/NVIDIA/gpu-operator
- Preferred installation method: helm
- Tags: ai, monitoring

**Reply by jakepage91** [reply]
The NVIDIA gpu-operator is not officially supported! Give it a go and let us know what you think.

**Comment by amorey** [comment]
* Package name: kubetail
* Short description: Real-time log viewer for Kubernetes
* Website URL: https://github.com/kubetail-org/kubetail
* Source code URL: https://github.com/kubetail-org/kubetail
* Logo URL: https://raw.githubusercontent.com/kubetail-org/kubetail/main/assets/kubetail-circle-icon.svg
* Preferred installation method: helm
* Tags: logging, monitoring

**Reply by jakepage91** [reply]
Kubetail is officially supported!

**Comment by Niek** [comment]
Package name: caddy-ingress
Short description: Kubernetes Ingress Controller for Caddy. It includes functionality for monitoring Ingress resources on a Kubernetes cluster and includes support for providing automatic HTTPS certificates for all hostnames defined in the ingress resources that it is managing.
Website URL: https://github.com/caddyserver/ingress
Source code URL: https://github.com/caddyserver/ingress
Logo URL: https://avatars.githubusercontent.com/u/12955528
Preferred installation method: helm
Tags: networking

**Reply by jakepage91** [reply]
Caddy ingress is officially supported!
